src/runtime_src/core/edge/drm/zocl/zocl_xgq.c

```diff
@@ -133,6 +133,14 @@ static int zxgq_fetch_request(struct zocl_xgq *zxgq, struct xgq_cmd_sq_hdr **cmd
 		zxgq_err(zxgq, "Payload size %dB is too big, truncated!", (*cmd)->count);
 	}
 	cpy_fromio(buf + header_sz, cmd_addr + header_sz, cnt);
+#if 0
```

Comment:
hmm....

Reply:
I will add zocl XGQ debug capability in another pull request.